### PR TITLE
feat(integrations): add support for manatal

### PR DESCRIPTION
**Pull request by hilmia**
## Describe your changes
- Added configuration for manatal in providers.yaml
- Added manatal to docs-v2.
- Added manatal for Supported API groups in mint.json.

This provider has been used successfully in local development to connect to Manatal after connecting and entering the API key using Nango.

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -1901,6 +1901,24 @@ mailchimp:
         response_type: code
     docs: https://docs.nango.dev/integrations/all/mailchimp
 #Untested configuration. Please reach out if you have a test account that we can use to test it.
+manatal:
+    auth_mode: API_KEY
+    categories:
+        - crm
```
Comment (hassan254-prog) [inline packages/shared/providers.yaml:1907]: You can add Manatal to the list of crm and hr integration categories docs.

File: docs-v2/integrations/all/manatal.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+- [Manatal API docs](https://developers.manatal.com/)
```
Comment (hassan254-prog) [inline docs-v2/integrations/all/manatal.mdx:22]: - [Manatal Authentication](https://developers.manatal.com/reference/authorization)
- [API rate limiting](https://developers.manatal.com/reference/rate-limiting)
```suggestion
- [Manatal API docs](https://developers.manatal.com/)
```

Comment (hassan254-prog) [inline docs-v2/integrations/all/manatal.mdx:24]:
```suggestion
- [Manatal Authentication](https://developers.manatal.com/reference/authorization)
- [API rate limiting](https://developers.manatal.com/reference/rate-limiting)
- [Manatal API docs](https://developers.manatal.com/)
```

**Comment by hassan254-prog** [comment]
@hilmia thanks a lot for your contribution, just minor comments otherwise everything looks good.
You can update the logo with the following [svg](https://codefile.io/f/nswhObr7Ja) file.

**Review by bodinsamuel — APPROVED** [review]
Thanks for your contribution, we can merge after you addressed @hassan254-prog comment 👌🏻

**Comment by hilmia** [comment]
> @hilmia thanks a lot for your contribution, just minor comments otherwise everything looks good. You can update the logo with the following [svg](https://codefile.io/f/nswhObr7Ja) file.

Hi @hassan254-prog, thanks for the feedback. Could you clarify where exactly I should add the logo? I couldn’t find any reference to it in the documentation or codebase.

**Comment by hilmia** [comment]
Thanks for the feedback. Committed the changes

**Review by hassan254-prog — APPROVED** [review]

**Pull request merged**